modules/react/_examples/stories/examples/AriaLiveRegions/HiddenLiveRegion.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import {AccessibleHide, AriaLiveRegion} from '@workday/canvas-kit-react/common';
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
+import {FormField} from '@workday/canvas-kit-preview-react/form-field';
+import {Flex} from '@workday/canvas-kit-react/layout';
+import {Text} from '@workday/canvas-kit-react/text';
+import {TextInput} from '@workday/canvas-kit-react/text-input';
+
+let liveRegionStr = '';
+
```

Comment:
We should have the styled on the `Flex` wrapper up here using `createStyles` or `createStencil` for consistency.

Reply:
I know it's not many styles but, this should be added here since we are moving away from style props:

```
alignItems="center"
gap="s"